native/acter/src/api/auth.rs

```diff
@@ -106,32 +106,28 @@ pub async fn login_with_token_under_config(
     } = serde_json::from_str(&restore_token).context("Deserializing Restore Token failed")?;
     let user_id = session.user_id.to_string();
     // First we need to log in.
-    RUNTIME
-        .spawn(async move {
-            let client = config
-                .homeserver_url(homeurl)
-                .build()
-                .await
-                .context("building client from config failed")?;
-            client
-                .restore_session(session)
-                .await
-                .context("restoring failed")?;
-            let state = ClientStateBuilder::default()
-                .is_guest(is_guest)
-                .build()
-                .context("building client state builder failed")?;
-            let c = Client::new(client.clone(), state)
-                .await
-                .context("Couldn't create client")?;
-            info!(
-                "Successfully logged in user {:?}, device {:?} with token.",
-                user_id,
-                client.device_id(),
-            );
-            Ok(c)
-        })
-        .await?
+    let client = config
```

Comment:
`login_with_token()` calls `new_client_config()` and `login_with_token_under_config()`.
Because `login_with_token_under_config()` is wrapped by `RUNTIME.spawn()` now, I couldn't wrap `login_with_token()`, caller of `login_with_token_under_config()` by `RUNTIME.spawn()`.
As we know, `new_client_config()` contains async api of matrix and it must be spawned in tokio task.
So I wrapped `new_client_config()` by `RUNTIME.spawn()`.